extensions/search/pkg/search/index/index.go

```diff
@@ -232,10 +239,29 @@ func (i *Index) Search(ctx context.Context, req *searchsvc.SearchIndexRequest) (
 }
 
 // BuildMapping builds a bleve index mapping which can be used for indexing
-func BuildMapping() mapping.IndexMapping {
+func BuildMapping() (mapping.IndexMapping, error) {
+	NameMapping := bleve.NewTextFieldMapping()
+	NameMapping.Analyzer = "lowercaseKeyword"
```

Comment:
Maybe Lowercase naming for `nameMapping`? Or you have a special reason for uppercase?